.drone.star

```diff
@@ -219,4 +224,112 @@ def notify():
         'failure'
       ]
     },
-  }
\ No newline at end of file
+  }
+
+def tests(ctx):
+  return {
+    'kind': 'pipeline',
+    'type': 'docker',
+    'name': 'tests',
+    'platform': {
+      'os': 'linux',
+      'arch': 'amd64',
+    },
+    'steps': [
+      {
+        'name': 'install',
+        'image': 'owncloudci/nodejs:11',
+        'pull': 'always',
+        'commands': [
+          'yarn install --frozen-lockfile',
+        ],
+      },
+      {
+        'name': 'lint',
+        'image': 'owncloudci/nodejs:11',
+        'pull': 'always',
+        'commands': [
+          'yarn lint',
+        ],
+      },
+      {
+        'name': 'build',
+        'image': 'owncloudci/nodejs:11',
+        'pull': 'always',
+        'commands': [
+          'yarn build',
+        ],
+      },
+    ],
+    'trigger': {
+      'ref': [
+        'refs/heads/master',
+        'refs/tags/**',
+        'refs/pull/**',
+      ],
+    },
+  }
+
+def release(ctx):
+  return {
+    'kind': 'pipeline',
+    'type': 'docker',
+    'name': 'release',
+    'platform': {
+      'os': 'linux',
+      'arch': 'amd64',
+    },
+    'steps': [
+      {
+        'name': 'build',
+        'image': 'owncloudci/nodejs:11',
+        'pull': 'always',
+        'commands': [
+          'yarn install --frozen-lockfile',
+          'yarn build'
+        ]
+      },
+      {
+        'name': 'publish-github',
+        'image': 'plugins/github-release:latest',
+        'pull': 'always',
+        'settings': {
+          'api_key': {
+            'from_secret': 'github_token',
+          },
+          'title': ctx.build.ref.replace("refs/tags/v", ""),
+          'note': 'dist/CHANGELOG.md',
```

Comment:
You need to build the changelog first. The other alrready built changelog is from another pipeline. Also  you need only the changes for that specific tag. Example in https://github.com/owncloud/ocis/blob/master/.drone.star#L565

Reply:
I see, thanks! I added it.